packages/logo-react/src/Logo.tsx

```diff
@@ -29,8 +29,20 @@ export const Logo = ({ className, centered = true, isSymbol, animated, title = "
                 viewBox={!animated && isSymbol && !centered ? "31 60 150 83" : "32 27 581 120"}
             >
                 <title id={uniqueId}>{title}</title>
-                <g fill="none" className="jkl-logo__F" strokeWidth="8">
-                    <line x1="34.3" x2="104.846844" y1="32.563024" y2="32.563024" className="jkl-logo__F-line" />
+                <g
+                    fill="none"
+                    className="jkl-logo__F"
+                    strokeWidth="8"
+                    transform={`${isSymbol ? "rotate(45) translate(90, -70)" : ""}`}
+                >
+                    <line
+                        x1="34.3"
+                        x2="104.846844"
+                        y1="32.563024"
+                        y2="32.563024"
+                        className="jkl-logo__F-line"
```

Comment:
Trengs klassen enda da? Burde vel kunne fjerne transform fra klasse hvertfall. Good job med å finne ut av dette for IE11 btw!

Reply:
`jkl-logo__F-line` må være med for å gi transition i normalt fungerende browsere. Det får man ikke gjennom attributten.